diff --git a/docroot/sites/all/modules/contrib/youtube/youtube.theme.inc b/docroot/sites/all/modules/contrib/youtube/youtube.theme.inc
index 803ba1c..5c5730a 100644
--- a/docroot/sites/all/modules/contrib/youtube/youtube.theme.inc
+++ b/docroot/sites/all/modules/contrib/youtube/youtube.theme.inc
@@ -34,7 +34,6 @@ function theme_youtube_video($variables) {
 
   // Protocol changes based on current page TODO.
   $protocol = (isset($_SERVER['HTTPS'])) ? 'https' : 'http';
-
   // Query string changes based on setings.
   $query = array();
   if (!$suggest) {
@@ -76,7 +75,8 @@ function theme_youtube_video($variables) {
   // Domain changes based on settings.
   $domain = ($privacy) ? 'youtube-nocookie.com' : 'youtube.com';
 
-  $path = $protocol . '://www.' . $domain . '/embed/' . $id;
+  //$path = $protocol . '://www.' . $domain . '/embed/' . $id;
+  $path = '//www.' . $domain . '/embed/' . $id;
   $src = url($path, array('query' => $query));
 
   $output = '<iframe id="' . drupal_html_id($player_class) . '" class="' . $player_class . '"
