diff --git a/docroot/sites/all/modules/contrib/addtoany/addtoany.module b/docroot/sites/all/modules/contrib/addtoany/addtoany.module
index 3c760cb..ea4fc22 100644
--- a/docroot/sites/all/modules/contrib/addtoany/addtoany.module
+++ b/docroot/sites/all/modules/contrib/addtoany/addtoany.module
@@ -142,8 +142,9 @@ function addtoany_page_alter(&$page) {
 	if ( ! $_addtoany_init)
 		return;
 	
-	$script_url = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on') ? 'https://static.addtoany.com/menu/page.js' : 'http://static.addtoany.com/menu/page.js';
-	
+	// $script_url = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on') ? 'https://static.addtoany.com/menu/page.js' : 'http://static.addtoany.com/menu/page.js';
+	$script_url = 'https://static.addtoany.com/menu/page.js';
+
 	$javascript_header = "var a2a_config=a2a_config||{},"
 			. "da2a={done:false,"
 			. "html_done:false,"
